params/config.go

```diff
@@ -33,15 +33,6 @@ var (
 	GoerliGenesisHash  = common.HexToHash("0xbf7e331f7f7c1dd2e05159666b3bf8bc7a8a3a9eb1d518969eab529dd9b88c1a")
 )
 
-// TrustedCheckpoints associates each known checkpoint with the genesis hash of
-// the chain it belongs to.
-var TrustedCheckpoints = map[common.Hash]*TrustedCheckpoint{
-	MainnetGenesisHash: MainnetTrustedCheckpoint,
-	TestnetGenesisHash: TestnetTrustedCheckpoint,
-	RinkebyGenesisHash: RinkebyTrustedCheckpoint,
-	GoerliGenesisHash:  GoerliTrustedCheckpoint,
-}
```

Comment:
Not sure that we should remove this. I'd stick with `nil` being the default trusted checkpoint, in the case of which we use this map to figure out the correct one. This keeps the initialization logic simpler, which is beneficial especially for library users. I'd actually do this same thing for the checkpoint oracles too.